astropy/utils/tests/test_introspection.py

```diff
@@ -92,3 +92,26 @@ def a_staticmethod(): pass
     assert not isinstancemethod(MyClass, MyClass.another_classmethod)
     assert not isinstancemethod(MyClass, MyClass.a_staticmethod)
     assert isinstancemethod(MyClass, MyClass.an_instancemethod)
+
+
+def _minversion_test():
```

Comment:
No. `_minversion_test()` is called twice already, at least on systems that have `pkg_resources`. Once inside the `if` clause to trigger an `ImportError` if `pkg_resources` is present to fall back on `distutils` and once after the `if` clause to use `pkg_resources`.